docs/specification/1.0.md

```diff
@@ -523,6 +524,36 @@ The following example illustrates a delete action:
 }
 ```
 
+
+
+#### overrideReason
+
+An **overrideReason** is described by the following attributes. No standard set of **overridereasons** exists today; implementers are encouraged to submit suggestions for standardization. To enable analysis of a card's usage, the CDS service SHALL send the same `code`, `codesystem`, and `label` across CDS Hooks requests for the same CDS client.
+
+Field | Optionality | Type | Description
+----- | ----- | ----- | --------
+`code` | REQUIRED | *string* | A code for this **overridereason**. `code` MUST be unique within the card.  To enable analysis of a card's usage, the CDS service SHALL send the same `code` accompanied by the same `label` across CDS Hooks requests for the same CDS client.
+`system` | OPTIONAL | *string* | A codesystem for this **overridereason** `code`.
+`label` | REQUIRED | *string* | A short, human-readable label to display for this override (e.g. the CDS Client might render this as the text on a button).
+
+
+```json
+{
+   "ackReasons":[
+      {
+         "code":"reason-id-provided-by-service",
+	 "system":"urn:oid:1.2.840.46474.1.3.27.11.6.4.373483.0",
```

Comment:
Should use a code system URL here, not an OID... suggest a made up one: `http://example.org/cds-services/fhir/CodeSystem/override-reasons`